umi_tools/umi_methods.py

```diff
@@ -351,9 +401,15 @@ def get_bundles(insam, read_events,
                     reads_dict[pos][key][umi]["read"] = read
 
     # yield remaining bundles
+    for x in reads_dict:
+        U.info(reads_dict[x])
+        for y in reads_dict[x]:
+            U.info(x)
+            U.info(y)
```

Comment:
Why this output? Seems like if there were quite a lot of reads on the last contig, and you are in per-contig or per-gene, this is going to output a whole lot of unneccesary crap